src/certifier.mak

```diff
@@ -56,9 +56,17 @@ LINK=g++
 # PROTO=/usr/local/bin/protoc
 PROTO=protoc
 AR=ar
+LL = ls -aFlrt
 
-# Definitions needed for generating Python bindings
+# Definitions needed for generating Python bindings using SWIG tool
 SWIG=swig
+
+# -Wallkw: Enable keyword warnings for all the supported languages
+SWIG_FLAGS = -Wallkw
+
+# Base of Certifier Framework's interface file for use by SWIG
+SWIG_CERT_INTERFACE = certifier_framework
```

Comment:
Cleaned up make file rules to consistently flag-off of single symbol, `SWIG_CERT_INTERFACE`.